app/controllers/books_controller.rb

```diff
@@ -40,18 +37,11 @@ def create
   end
 
   def update
-    @book.authors = []
-    
-    author_params[:author_ids].each do |k, v|
```

Comment:
[Suggestion]
We can permit the params `author_ids` and don't need to do anything. Just update new attributes